bpmsuite/bpmsuite70-executionserver-postgresql-persistent.json

```diff
@@ -0,0 +1,680 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "description": "Application template for Red Hat JBoss BPM Suite 7.0 which includes an execution server with a PostgreSQL.",
+            "iconClass": "icon-jboss",
+            "tags": "bpmsuite,jboss,xpaas",
+            "version": "1.0.0",
+            "openshift.io/display-name": "Red Hat JBoss BPM Suite 7.0 Execution Server + PostgreSQL (persistent)"
+        },
+        "name": "bpmsuite70-executionserver-postgresql-persistent"
+    },
+    "labels": {
+        "template": "bpmsuite70-executionserver-postgresql-persistent",
+        "xpaas": "1.4.0"
+    },
+    "message": "A new persistent BPM Suite Execution Server application (using PostgreSQL) has been created in your project. For accessing the PostgreSQL database \"${DB_DATABASE}\" use the credentials \"${DB_USERNAME}/${DB_PASSWORD}\".",
+    "parameters": [
+        {
+            "displayName": "EAP Admin User",
+            "description": "EAP administrator username",
+            "name": "ADMIN_USERNAME",
+            "value": "eapadmin",
+            "required": false
+        },
+        {
+            "displayName": "EAP Admin Password",
+            "description": "EAP administrator password",
+            "name": "ADMIN_PASSWORD",
+            "from": "[a-zA-Z]{6}[0-9]{1}!",
+            "generate": "expression",
+            "required": false
+        },
+        {
+            "displayName": "KIE Admin User",
+            "description": "KIE administrator username",
+            "name": "KIE_ADMIN_USER",
```

Comment:
KIE Admin User/password isn't needed here. Kie server uses KIE_SERVER_CONTROLLER_USER/KIE_SERVER_CONTROLLER_PWD for connecting to the Workbench.